addons/skin.estuary/xml/MyPlaylist.xml

```diff
@@ -1,163 +1,96 @@
-							<visible>Control.HasFocus(50)</visible>
-						</control>
-						<control type="label">
-							<left>30</left>
-							<bottom>0</bottom>
-							<right>30</right>
-							<aligny>center</aligny>
-							<font>font12</font>
-							<label>$VAR[PlaylistLabelVar]</label>
-							<shadowcolor>text_shadow</shadowcolor>
-						</control>
-						<control type="label">
-							<left>30</left>
-							<bottom>0</bottom>
-							<right>30</right>
-							<align>right</align>
-							<aligny>center</aligny>
-							<font>font12</font>
-							<label>$VAR[PlaylistLabel2Var]</label>
-							<shadowcolor>text_shadow</shadowcolor>
-						</control>
-					</focusedlayout>
-					<itemlayout height="list_item_height">
-						<control type="label">
-							<left>30</left>
-							<bottom>0</bottom>
-							<right>30</right>
-							<aligny>center</aligny>
-							<font>font12</font>
-							<label>$VAR[PlaylistLabelVar]</label>
-							<shadowcolor>text_shadow</shadowcolor>
-						</control>
-						<control type="label">
-							<left>30</left>
-							<bottom>0</bottom>
-							<right>30</right>
-							<align>right</align>
-							<aligny>center</aligny>
-							<font>font12</font>
-							<label>$VAR[PlaylistLabel2Var]</label>
-							<textcolor>grey</textcolor>
-							<shadowcolor>text_shadow</shadowcolor>
-						</control>
-					</itemlayout>
-				</control>
-				<control type="group">
-					<depth>DepthContentPanel</depth>
-					<include content="ContentPanel">
-						<param name="width" value="470" />
-					</include>
-					<control type="grouplist" id="700">
-						<orientation>vertical</orientation>
-						<itemgap>0</itemgap>
-						<left>0</left>
-						<top>120</top>
-						<onup>700</onup>
-						<ondown>700</ondown>
-						<onleft>50</onleft>
-						<onright>50</onright>
-						<width>410</width>
-						<control type="radiobutton" id="20">
-							<width>410</width>
-							<height>110</height>
-							<align>left</align>
-							<aligny>top</aligny>
-							<texturefocus colordiffuse="button_focus">lists/focus.png</texturefocus>
-							<texturenofocus />
-							<radioposx>300</radioposx>
-							<textoffsetx>40</textoffsetx>
-							<textoffsety>35</textoffsety>
-							<label>$LOCALIZE[191]</label>
-						</control>
-						<include content="PlaylistWindowButton">
-							<param name="control_id" value="26" />
-							<param name="label" value="" />
-						</include>
-						<include content="PlaylistWindowButton">
-							<param name="control_id" value="21" />
-							<param name="label" value="$LOCALIZE[190]" />
-						</include>
-						<include content="PlaylistWindowButton">
-							<param name="control_id" value="22" />
-							<param name="label" value="$LOCALIZE[192]" />
-						</include>
-						<include content="MediaMenuNowPlaying">
-							<param name="left" value="-5" />
-						</include>
-					</control>
-				</control>
+				<visible>Control.IsVisible(55) | Control.IsVisible(503)</visible>
```

Comment:
this condition isn't needed i think?
it will always return true since those are the only two views available

Reply:
The condition is necessary as the Now Playing view is only used for music playlists. Video playlists only have the single Widelist view with this condition in place.

Reply:
ok, tested it. removing this condition makes no difference at all.
can you please remove it at your end as well and provide a screenshot of what goes wrong?

Reply:
Apologies I misunderstood which line you were talking about, will take another look now I've realised it's L16